docs/index.json

```diff
@@ -4,7 +4,7 @@
odules from: /home/michael/go/pkg/mod/github.com/gokrazy/kernel@v0.0.0-20230111172439-0cd82b0bec82/lib/modules Feature summary: use GPT: true use PARTUUID: true use GPT PARTUUID: true 2023/01/12 23:36:36 partitioning /dev/disk/by-id/usb-TS-RDF5_SD_Transcend_000000000037-0:0 (GPT + Hybrid MBR) 2023/01/12 23:36:36 device holds 15931539456 bytes 2023/01/12 23:36:36 Re-reading partition table failed: permission denied. Remember to unplug and re-plug the SD card before creating a file system for persistent data, if desired. Creating boot file system [creating boot file system] Kernel directory: /home/michael/go/pkg/mod/github.com/gokrazy/kernel@v0.0.0-20230111172439-0cd82b0bec82 EEPROM update summary: pieeprom.upd (sig d1e44edf33) recovery.bin vl805.bin (sig 6246230ecd) [done] in 0.18s, 71 MiB MBR summary: LBAs: vmlinuz=51795 cmdline.txt=144875 PARTUUID: 4f9f2cab Creating root file system [done] in 0.33s If your applications need to store persistent data, unplug and re-plug the SD card, then create a file system using e.g.: mkfs.ext4 /dev/disk/by-partuuid/60c24cc1-f3f9-427a-8199-4f9f2cab0004 To boot gokrazy, plug the SD card into a supported device (see https://gokrazy.org/platforms/) Build complete! To interact with the device, gokrazy provides a web interface reachable at: http://gokrazy:\u0026lt;automatically-generated-random-password\u0026gt;@hello/ In addition, the following Linux consoles are set up: 1. foreground Linux framebuffer console on HDMI It is safe to unplug your SD card once gok returns.\nStep 3: Boot gokrazy After booting from this SD card, your device will:\ndisplay system status on the monitor connected via HDMI, if any obtain an IP address for hostname “gokrazy” via DHCP (IPv4) and SLAAC (IPv6) synchronize the clock using NTP expose a password-authenticated web interface on private IP addresses\n(each instance gets its own password in the Update → HTTPPassword key of its config.json, see gok edit) supervise all installed programs (only the hello world program in this example) To interact with your device, you can:\nOpen the gokrazy web interface in your browser (easiest option) Attach a serial console (see below) Use the breakglass package to start an interactive SSH session. gok new sets it up by default. Congratulations! 🎉 You now have a working gokrazy installation!\nNext, see the Configuration guide for details on how to use other programs.\nOptional: Using the serial console See also: Configuration → Instance Config Reference → SerialConsole\nIf you prefer the serial console, you can optionally connect a serial adapter to the Raspberry Pi and remove the \u0026quot;SerialConsole\u0026quot;: \u0026quot;disabled\u0026quot; line from your gokrazy instance’s config.json to make the primary Linux console end up on the serial console.\nWhen pressing Enter on the serial console, gokrazy will interactively start either:\n/tmp/serial-busybox/ash (provided by the serial-busybox package) /perm/sh, if present (supplied by you, the user) "
```

Comment:
Can you revert your change to this index file please? I’ll update it as part of the website regeneration step.